connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -597,7 +596,9 @@ private Set<TopicPartition> listPartitions(
             Admin admin,
             Collection<String> topics
     ) throws TimeoutException, InterruptedException, ExecutionException {
-        assertFalse("collection of topics may not be empty", topics.isEmpty());
```

Comment:
Removed assertions from this API as it added an implicit dependency on JUnit 4, which we don't use in the `:connect:mirror` module.